tests/test_matrix_props/test_is_permutation.py

```diff
@@ -0,0 +1,42 @@
+"""Test is_permutation."""
+import numpy as np
+
+from toqito.matrix_props.is_permutation import is_permutation

```

Comment:
This import statement is incorrect. Look at the other files for example. It should be 

```
from toqito.matrix_props import is_permutation
```

In order for that to work, you need to include 

```
from toqito.matrix_props.is_permutation import is_permutation
```

inside the `__init__.py` of the `matrix_props` directory.

Reply:
Thanks so much,.. I was wondering why I could not import it like the other examples.  Will check out the __init__.py file and correct.